README.md

```diff
@@ -28,7 +28,13 @@ For functional testing, please see the [functional testing documentation](test/d
 
 When you attempt to commit code changes, several of the above tasks will be run automatically to help ensure that your changes pass tests and are consistent with agreed-upon standards.
 
-Thank you for your interest in contributing! 
+Thank you for your interest in contributing!
+
+## Linking to Documentation
+
+If you are writing code that logs a message referring the user to a particular piece of [documentation](https://app.gitbook.com/@launch/s/adobe-experience-platform-web-sdk), please link to the documentation using an `adobe.ly` URL. To make an `adobe.ly` URL, go to [Bitly](https://bitly.com/) and shorten the documentation URL. Bitly should provide you with a short url using the `adobe.ly` domain.
```

Comment:
Do you mean bit.ly here when saying adobe.ly?

Reply:
I did mean bit.ly. Am I missing something? As far as I'm aware, I'm unable to make an adobe.ly URL by going to adobe.ly.

Reply:
Oh wait. I read your comment wrong. Yes, I meant adobe.ly when I said adobe.ly. Is it wrong for some reason?